%HEADER
\documentclass[12pt,letterpaper]{article}

%packages
\usepackage{fixltx2e}
\usepackage{textcomp}
\usepackage{fullpage}
\usepackage{amsfonts}
\usepackage{verbatim}
\usepackage[english]{babel}
\usepackage{pifont}
\usepackage{color}
\usepackage{setspace}
\usepackage{lscape}
\usepackage{indentfirst}
\usepackage[normalem]{ulem}
\usepackage{booktabs}
%\usepackage{nag}
\usepackage{natbib}
%\usepackage{bibtex}
\usepackage{float}
\usepackage{latexsym}
%\usepackage{hyperref} 
\usepackage{url}
%\usepackage{html}
\usepackage{hyperref}
\usepackage{epsfig}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{amsmath}
\usepackage{bm}
\usepackage{array}
\usepackage[version=3]{mhchem}
\usepackage{ifthen}
\usepackage{caption}
\usepackage{hyperref}
%\usepackage{xcolor}
\usepackage{amsthm}
\usepackage{amstext} % Add and remove packages as necessary for your manuscript.

%Pagination style and stuff
\linespread{1.66} % All text should be double-spaced with occasional exceptions for tables. 
\raggedright
\setlength{\parindent}{0.5in}

\setcounter{secnumdepth}{0} % Our sections are not numbered and our papers do not have Tables of Contents. We don't present a list of figures or list of tables, either. Any common font is fine. (A common sans-serif font should be used on figures, but figures should be separate from the LaTeX document.)

\pagestyle{empty}

\renewcommand{\section}[1]{%
\bigskip
\begin{center}
\begin{Large}
\normalfont\scshape #1
\medskip
\end{Large}
\end{center}}

\renewcommand{\subsection}[1]{%
\bigskip
\begin{center}
\begin{large}
\normalfont\itshape #1
\end{large}
\end{center}}

\renewcommand{\subsubsection}[1]{%
\vspace{2ex}
\noindent
\textit{#1.}---}

\renewcommand{\tableofcontents}{}

\bibpunct{(}{)}{;}{a}{}{,}  % this is a citation format command for natbib
% We don't use a special title page; the author information is entered like any other text.
% FOOTNOTES: We don't allow them in the manuscript, except in tables. Don't include any footnotes in the text.
%IMPORTANT:
%Note on display in the text file: sentences or sub-parts of sentences (e.g. enumerations) are delimited by a newline for clarity.
%However, true new lines (i.e. in the pdf) are delimited by one or more empty lines.
%We don't use a heading for the introduction. The nature of this section is implied by its position following the abstract and preceding the first major section heading.



%---------------------------------------------
%
%       START
%
%---------------------------------------------

\begin{document}

%Runing head
\begin{flushright}
Version dated: \today
\end{flushright}
\bigskip
\noindent RH: Missing data and topology in total evidence matrices %RH = Running Head

\bigskip
\medskip
\begin{center}

%Title
\noindent{\Large \bf Effect of missing data in matrices containing living and fossil taxa on topological accuracy}
%Alternative punch title: "Bayesian method failure to infer topology using matrices containing both living and fossil taxa"
%Alternative punch title: "ML outperforms Bayesian for inferring topology from matrices containing both living and fossil taxa"
\bigskip

%Authors
\noindent {\normalsize \sc Thomas Guillerme$^1$$^,$$^2$, other authors $^3$, and Natalie Cooper$^1$$^,$$^2$}\\
\noindent {\small \it 
$^1$School of Natural Sciences, Trinity College Dublin, Dublin 2, Republic of Ireland;\\
$^2$Trinity Centre for Biodiversity Research, Trinity College Dublin, Dublin 2, Republic of Ireland;\\
$^3$Somewhere else }\\
\end{center}
\medskip
\noindent{\bf Corresponding author:} Thomas Guillerme, School of Natural Sciences, Trinity College Dublin, Dublin 2, Republic of Ireland; E-mail: guillert@tcd.ie.\\
\vspace{1in}


%---------------------------------------------
%
%       ABSTRACT
%
%---------------------------------------------


\subsubsection{Abstract}
Living species represent less than 1\% of all species that have ever lived. Ignoring fossil taxa may lead to misinterpretation of macroevolutionary patterns and processes such as trends in species richness, biogeographical history or paleoecology.
This fact has led to an increasing consensus among scientists that both fossil and living taxa must be included in macroevolutionary studies.
One approach, the total evidence approach, uses molecular data from living taxa and morphological data from both living and fossil taxa to infer phylogenies with both fossil and living taxa at the tips.
Although the total evidence approach seems very promising, it requires a lot of data and is therefore likely to suffer from missing data issues which may affect its ability to infer correct phylogenies.

In this study we assess the effect of missing data on tree topologies inferred from total evidence matrices.
Using simulations we investigate three major factors that directly affect the completeness of the morphological part of the matrix:
(1) the proportion of living taxa with no morphological data,
(2) the amount of missing data in the fossil record and
(3) the overall number of morphological characters in the matrix.
We find that, in a Bayesian framework, difficulties in recovering a stable topology are mainly driven by the missing data in the molecular part of the matrix (for which fossil taxa have no data).
In a Maximum Likelihood framework, however, topology is not directly affected by missing data \textit{per se}, but by the number of morphological characters shared among the taxa.
Therefore, the two main drivers of incorrect topologies are the overall number of morphological characters and the number of living species with no morphological data.

Our results suggest that, in order to use total evidence approaches, one should reduce the missing data in the morphological part of the matrix for living species and use a Maximum Likelihood framework to fix the topology prior to the overall Bayesian phylogenetic inference process.
%More results!

\noindent (Keywords: missing data, total evidence, Bayesian, Maximum Likelihood, topology)\\

\vspace{1.5in}

\newpage 


%---------------------------------------------
%
%       INTRODUCTION
%
%---------------------------------------------


Living species represent less than 1\% of all species that have ever lived \citep{novacek1992ext,raup1993extinction}.
However, the majority of macroevolutionary studies focus solely on living species \citep{cooperwhat2009,meredithimpacts2011,healyecology2014}. %or is citing of your own papers in the too "douchebagy" for the second sentence?
Ignoring fossil taxa may lead to misinterpretation of macroevolutionary patterns and processes such as timing of diversification events \citep[e.g.][]{pyrondivergence2011}, relationships among lineages \citep[e.g.][]{manosphylogeny2007} or niche occupancy \citep[e.g.][]{pearmanniche2008}.
These factors have led to increasing consensus among scientists that fossil taxa must be included in macroevolutionary studies \citep{jacksonwhat2006,quentaldiversity2010,dietlconservation2011,slaterunifying2013,fritzdiversity2013}.
However, to do this we need to be able to place living and fossil taxa into the same phylogenies which still remains complex despite an increasing number of significant efforts \citep{pyrondivergence2011,ronquista2012,schragocombining2013}. %Not a nice sentence

Up to now, three main approaches have been used for combining fossil and living taxa data in phylogenies.
These approaches differ mainly in whether they treat fossil taxa as tips or as nodes in the phylogeny and on which part of the available data of the fossil is used (i.e. the fossil occurrence age only or both the age and the morphology).
Classical cladistic methods uses matrices containing morphological data from both fossil and living taxa and allows to treat each taxon as a tip in the phylogeny; the relation between the taxa can be inferred using optimal criteria such as the maximum parsimony criterion \citep{simpson1945}.
This approach is commonly used by paleontologists but it ignores the additional molecular data available from living species and doesn't allow to use probabilistic methods to deal with inference uncertainty (but see \citet{spencerefficacy2013}.
Neontologists, on the other hand, more commonly use probabilistic methods (e.g. Maximum Likelihood or Bayesian) based on matrices containing only molecular data from living species.
Because fossil taxa do not usually have available DNA, fossils are used as nodes rather than tips in these analyses and only their occurrence age can be used to time calibrate phylogenies \citep{zuckerkandl1965}.
There have been great improvements in the theory and application of these two approaches \citep[e.g.][]{bapsta2013,stadlerdating2013,heaththe2013} as well as much debate about the "best" approach to use \citep[e.g.][]{spencerefficacy2013}.
However none of them is optimal and uses all the available data.

A final approach, known as total evidence approach use matrices containing molecular data from living taxa and morphological data from both living and fossil taxa. %There might be a philosophico-semantic distinction to do here. Is TEM a method or an approach? I'll say that it is an approach because it's more like an idea (put every thing together) than a way to do (how to put everything together). It makes more sense to me compared to a real method e.g. a probabilistic method such as ML where the method states precisely how to do stuff.
It allows to use probabilistic methods, to treat every taxon as a tip in the phylogeny and to use the occurrence age of the fossil to time calibrate the phylogeny \citep{eernissetaxonomic1993}.
Here we focus on this total evidence approach because they have been recently successfully developed and applied to empirical data \citep{pyrondivergence2011,ronquista2012,schragocombining2013}.  %needs a better justification
Although the total evidence approach seems very promising, there is one big drawback in using this approach: it requires a lot of data.
In particular it requires morphological data form both living and fossil taxa, both of which are known to be scarce.
Therefore total evidence approach is likely to suffer from having lots of missing data which may affect their ability to infer correct topology, branch length and support values \citep{salamin2003}. % Trevor: Missing data also influences support values that trend to decrease

The effect of missing data on phylogenetic inferences has been widely studied \citep{wiensmissing2003,wiensmissing2006,wiensmissing2008,lemmonthe2009,rouresite-specific2011,sansomfossilization2013}.
Missing molecular data has been seen by some authors as an issue because it can decrease the phylogenetic signal in some parts of the tree, especially when using large matrices \citep{lemmonthe2009}.
However other authors do not see missing molecular data as a major issue because the phylogenetic signal, rather than by reducing the amount of missing data, is more likely to increase by having:
(i) at least a "modest" number of highly covered genes (i.e. approximatively half of the genes - \citet{rouresite-specific2011});
(ii) a higher number of taxa (especially slowly evolving taxa or taxa close to the outgroup - \citet{rouresite-specific2011}) and
(iii) by choosing more adequate models of sequence evolution rather than by reducing the amount of missing data \citep{wiensmissing2006,wiensmissing2008,rouresite-specific2011}.

Similarly, depending on the study, missing morphological data might be seen as either a major or minor issue for accurately inferring phylogenies depending on the study \citep{wiensmissing2003,sansomfossilization2013}.
Because soft-tissues characters are rarely preserved in the fossil record, missing data is mainly found in soft tissues characters, and is therefore not randomly distributed, which can lead to biased placement of fossil taxa in phylogenies \citep{sansomfossilization2013}.
However, the phylogenetic signal is not related to the level of missing data \textit{per se} but to the number of informative characters per taxa, therefore missing data is less an issue than the number of shared informative characters \citep{wiensmissing2003}.
Although missing data as been shown to be no major problem separately in molecular and morphological matrices \citep{wiensmissing2003,wiensmissing2006,wiensmissing2008,rouresite-specific2011} it may become an issue when combined in a total evidence type matrices.
Because the nature of total evidence matrices, combining both molecular and morphological information for both living and fossil taxa, a big part of the matrix contains only missing data - the molecular data for fossil taxa.
Until now, no attempt has been made to study the impact of this issue on phylogenetic inference and especially on topological recovery.

Here we assess the effect of missing data on tree topology inferred from total evidence matrices.
The molecular part for living taxa of a total evidence matrix acts like a "classical" molecular matrix \citep{ronquista2012}.
The effect of missing data on such matrices is well known \citep{wiensmissing2006,wiensmissing2008,rouresite-specific2011}, therefore, we only focus on the missing data issue in the morphological part of the matrix.
Using simulations we investigate three major factors that directly affect the completeness of the morphological part of the matrix:
(1) the proportion of living taxa with no morphological data,
(2) the amount of missing data in the fossil taxa (i.e. the preservation quality of the fossil record) and
(3) the overall number of morphological characters for both living and fossil taxa in the matrix.
We assess how removing data from a total evidence type matrix by changing the values of these three parameters affects the topology of the inferred tree using Maximum Likelihood and Bayesian method.
%This paragraph is crap

We found that whatever the amount of missing data, total evidence matrices dramatically reduce the performance of Bayesian inference.
However, when using a Maximum Likelihood approach, missing data decreases gradually the ability of recovering the right tree topology.
We propose that this drastic difference between Bayesian and Maximum Likelihood methods is due to a flattening of the likelihood landscape because of the intrinsic amount of missing data in a total evidence matrix (i.e. the missing molecular data for the fossil taxa).
This problem leads to an inability of Bayesian chains to pick up the sub and near optimal topologies leading to high variance in the posterior tree distribution and really low support in the consensus tree.
These results should encourage authors in not using Bayesian inference to explore the topological landscape of their phylogeny.
Instead, we suggest that authors should first find the Maximum Likelihood tree and use it's topology as a fix prior in their Bayesian inference.
We also propose a new method for comparing Bayesian posterior tree distribution's topology available in a wonderful R package. %Or am I just dreaming a bit too much here?

%---------------------------------------------
%
%       METHODS
%
%---------------------------------------------


\section{Methods} %no material needed?
To explore the effect of missing data on total evidence trees topologies we used the following protocol (note that we explain each step in detail below this general outline -Fig. ~\ref{Fig_Outline}):
\begin{enumerate}
\item{Generating the matrix:} \\
\label{step:generate_matrix}
We randomly generated birth death tree (hereafter called the "true" tree, Table ~\ref{Tab_glossary}) to infer a matrix containing both molecular and morphological data for living and fossil taxa (hereafter called the "complete" matrix, Table ~\ref{Tab_glossary}).
\item{Removing data:} \\
\label{step:remove_data}
We removed data from the morphological part of the "complete" matrix to simulate the effects of missing data by modifying three parameters (1) the proportion of missing living taxa ($M_{L}$), (2) the proportion of missing data in the fossil taxa ($M_{F}$) and (3) the proportion of missing morphological characters ($M_{C}$) (the resulting matrices are called hereafter the "missing-data" matrix, Table ~\ref{Tab_glossary}).
\item{Building phylogenies:} \\
\label{step:build_phylo}
We inferred Bayesian phylogenetic trees from the "complete" matrix and from the "missing-data" matrices resulting in respectively a tree generate from a matrix containing no missing data in the morphological part (hereafter called the "best" tree, Table ~\ref{Tab_glossary}) and trees inferred from matrices with missing data in their morphological part (hereafter called the "missing-data" trees, Table ~\ref{Tab_glossary}).
\item{Comparing topologies:} \\
\label{step:compare_topo}
We then compared the "best" tree to the "missing-data" trees to assess the influence of each parameter ($M_{L}$, $M_{F}$, $M_{C}$ and their interactions) on the topologies of the phylogenies we estimated.
\end{enumerate}
To measure the effect of missing data distribution, we repeated steps ~\ref{step:generate_matrix} to ~\ref{step:compare_topo} with the exact same fixed parameters 51 (3$\times$17) times. %that's because I had 3 cluster accounts available... If it's a too odd number, no worries, I'll just remove the last chain.
A list of all the terms used in this paper is available in table ~\ref{Tab_glossary}.


\begin{figure}
\centering
\includegraphics[keepaspectratio=true]{Figures/TEM_Fig_outline-BW.pdf}
\caption{Protocol outline.
(1) We generated a random tree (the "true" tree) to infer a matrix with no missing data (the "complete" matrix).
(2) We removed data fron the morphological partition of the "complete" matrix resulting in 124 "missing-data" matrices.
(3) We infer phylogenetic trees from each matrix in both ML and Bayesian framework.
(4) We then compared the "missing-data" trees to the "best" tree.
We repeated step 1 to 4 51 (3$\times $17) times.}
\label{Fig_Outline}
\end{figure}

\begin{table}[ht]
\caption{Glossary}
\centering
\begin{tabular}{p{5cm} p{9cm}}
\hline
\hfill Term & Definition \\[0.5ex]
\hline
\hfill living taxa & taxa with both molecular and morphological data available \\[1.5ex]
\hfill fossil taxa & taxa with only morphological data available \\[1.5ex]
\hfill "complete" matrix & matrix with no missing data except for the molecular part of the fossil taxa \\[1.5ex]
\hfill "missing-data" matrix & matrix with various amount of missing data \\[1.5ex]
\hfill $M_{L}$ & missing living taxa in the morphological part of the matrix \\[1.5ex]
\hfill $M_{F}$ & missing data for the fossil taxa in morphological part of the matrix \\[1.5ex]
\hfill $M_{C}$ & missing morphological characters for both living and fossil taxa \\[1.5ex]
\hfill "true" tree & tree used to simulate the matrix \\[1.5ex]
\hfill "best" tree & tree inferred from the "complete" matrix \\[1.5ex]
\hfill "missing-data" tree & tree inferred the "missing-data" matrices \\[1.5ex]
\hfill RPBTC & Random pairwise Bayesian tree comparison \\[1.5ex]
\hfill "horizontal" sub-matrix & sub-matrix with morphological and molecular characters for living taxa \\[1.5ex]
\hfill "vertical" sub-matrix & sub-matrix with morphological data for both living and fossil taxa \\[1.5ex]
\hfill "corner" sub-matrix & sub-matrix with morphological data for living taxa \\[1.5ex]
\hfill "missing-data" sub-matrix & molecular part of the matrix for the fossil taxa (no data) \\
\end{tabular}
\label{Tab_glossary}
\end{table}

%---------------------------------------------
%       Generating the matrix
%---------------------------------------------

\subsection{Generating the matrix}
First we randomly generated a "true" tree of 50 taxa in R v3.0.2 \citep{R302} using the package diversitree v0.9-6 \citep{fitzjohndiversitree2012}.
We generated the tree using a Birth Death process by sampling the values of the speciation events ($\lambda$) and extinction events ($\mu$) from a uniform distribution but maintaining $\lambda$ $>$ $\mu$ \citep{paradistime-dependent2011}.
We implemented a rejection sampling algorithm to select only random trees with 25 living and 25 fossil taxa. %Justification needed?
We then added a taxa to the resulting Birth-Death tree as the outgroup of the tree.
The mean branch length of the tree was used to separate the outgroup from the rest of the taxa and the branch length leading to the outgroup was set as the sum of the mean branch length and the longest root-to-tip length of the tree.

Next, we created a molecular and a morphological matrix from the "true" tree.
The molecular matrix was inferred from the "true" tree using the package phyclust v0.1-14 \citep{chen2011}.
The matrix was made of 1000 characters sites for 51 taxa and generated using the seqgen algorithm \citep{ranbaut1997seqgen}.
We used the HKY model \citep{HKY85} with a random base frequencies and with the transition/transversion rate of 2 \citep{douadycomparison2003} as parameters for generating the matrix.
The substitution rates were distributed following a gamma distribution with an alpha ($\alpha$) shape of 0.5 \citep{yangamong-site1996}.
We chose a low value of $\alpha$ to reduce the number of sites with high substitution rates, thus avoiding too much homoplasy and a decrease in phylogenetic signal.
These parameters were selected to generate data with no special assumption about how the characters evolved as well as to reduce the computational time required if these parameters were estimated rather than defined (total computational time $>$ 65 CPU years).

We inferred the morphological matrix using the ape package v3.0-11 \citep{paradisape:2004} to generate a matrix of 100 character sites for 51 taxa.
We assigned the number of character states (either 2 or 3) for each morphological character by sampling with a probability of 0.85 for two states characters and 0.15 for three state characters.
These probabilities were selected using the overall distribution of characters states extracted from 100 published empirical morphological matrices (See \hyperref[supplementaries]{supplementaries}).
We then ran an independent discrete character simulation for each character using the "true" tree branch length and topology with the randomly selected number of states (2 or 3) and assuming an equal rate of change (i.e. evolutionary rate) from one character state to an other \citep{Pagel22011994}.
This method allows us to have only two parameters per character: the number of states and the evolutionary rate.
For each character, the evolutionary rate was sampled from a gamma distribution with $\alpha$ = 0.5.
We used a low evolution rate parameter (i.e. $\alpha$) in order to avoid homoplasy in the morphological part of the matrix and create a clear phylogenetic signal \citep{wagner2000,davalosintegrating2014}.

All the molecular information for fossil taxa was replaced by missing data ("?").
Finally, we combined the morphological and molecular matrices obtained from the "true" tree.
Hereafter we call this the "complete" matrix: the matrix with no missing data except for the molecular data of the fossil taxa.

%---------------------------------------------
%       Removing data
%---------------------------------------------

\subsection{Removing data}
Once we obtained the "complete" matrix we modified it to get a set of matrices with missing data. %Ugly sentence
We randomly replaced data with "?" in the morphological part of the matrices according to the following parameters (Fig. ~\ref{Fig_RemoveData}):

\begin{enumerate}
\item{The proportion of living taxa with no morphological data ($M_{L}$): 0\%, 10\%, 25\%, 50\% or 75\%.}
This parameter illustrates the number of living taxa that are present in the molecular part of the matrix but not in the morphological one.
Because of the increasing facility to sequence DNA for living taxa, the number of living taxa with molecular data is highly superior the the number of taxa with molecular and morphological data.
\item{The proportion of missing morphological data across all fossil taxa ($M_{F}$): 0\%, 10\%, 25\%, 50\% or 75\%.}
This parameter illustrates the quality of the fossil record. 
\item{The proportion of missing morphological characters across all taxa (living and fossil - $M_{C}$): 0\%, 10\%, 25\%, 50\% or 75\%. }
This parameter illustrates the number of available morphological characters for both living and fossil taxa.
\end{enumerate}

\begin{figure}
\centering
\includegraphics[keepaspectratio=true]{Figures/TEM_Fig_missingData-BW.pdf}
\caption{Different parts of the matrix. %Title?
A: Missing data parameters:
Missing living - The proportion of living taxa with no morphological data ($M_{L}$);
Missing fossil - The proportion of missing morphological data across all fossil taxa ($M_{F}$);
Missing character - The proportion of missing morphological characters across all taxa (living and fossil) ($M_{C}$).
B: Different parts of the matrix:
The "horizontal" sub-matrix (orange) contains molecular and morphological data for living taxa;
The "vertical" sub-matrix (blue) contains morphological data for living and fossil taxa;
The "corner" sub-matrix (orange and blue striped) contains morphological data for living taxa;
The "missing-data" sub-matrix (grey) is the molecular part of the fossil taxa and contains no data.}
\label{Fig_RemoveData}
\end{figure}

In practice, each parameter represent a different way of removing data in the morphological part of the matrix: $M_{L}$ removes a proportion of rows from the living taxa; $M_{F}$ removes a proportion of cells from the fossil taxa; and $M_{C}$ removes a proportion of columns across both living and fossil taxa.
Note that $M_{L}$ is different to $M_{F}$ not only because of the region of the matrix affected: for $M_{L}$, all the morphological data of a proportion of the living taxa is removed (i.e. removing rows), as for $M_{F}$, a proportion of data is removed across the whole of the morphological matrix for fossil taxa (i.e. removing cells).

We tested all parameters combinations resulting in 125 ($5^3$) matrices.
Because some parameter combinations introduce a lot of missing (e.g. $M_L$=75\%, $M_F$=75\% and $M_C$=75\%), some matrices contained fossil taxa without any data at all.
When this occurred we repeated the random deletion of characters until every taxa had at least 5\% data across the whole matrix.

%---------------------------------------------
%       Building phylogenies
%---------------------------------------------

\subsection{Building phylogenies}
From the resulting matrices we generated two types of trees, the "best" tree that is inferred from the "complete" matrix and the "missing-data" trees inferred from the 125 matrices with various amounts of missing data.
The "true" tree was used to generate the "complete" matrix and reflects the "true" evolutionary history in our simulations.
The "best" tree, on the other hand, is the best tree we can build using the state-of-the-art phylogenetic methods.
In real world situations, the "true" tree is never available to us because we cannot know the true evolutionnary history of a clade (except in very rare circumstances, e.g. \citet{rozen2005}).
Therefore, here we focus on comparing the trees inferred from the matrices with missing data to the "best" tree, rather than the "true" tree, as the "best" tree is generally what biologists have to work with.

\subsubsection{Maximum Likelihood}
The "best" tree and the "missing-data" trees were inferred using RAxML v8.0.20 \citep{Stamatakis21012014}.
For the molecular data, we used the GTR + $\Gamma_4$ model (\citet{tavare1986}; default GTRGAMMA in RAxML v8.0.20; \citet{Stamatakis21012014}) as a generalisation of the HKY + $\Gamma_4$ model \citep{HKY85} for the molecular data.
The GTR model can be seen as a generalisation of the HKY model (the 2 parameters from the HKY model are implicitly included in the 6 from GTR model - \citet{stamatakisa2008}).
For the morphological data, we used the implemented Markov \textit{k} state model \citep{lewisa2001} wich is a generalisation of the JC69 model \citep{jc69} with \textit{k} $\geq$ 2 assuming an equal state frequency and a unique overall substitution rate ($\mu$) following a gamma distribution of the rate variation with four distinct categories (M\textit{k} + $\Gamma_4$; -K MK option in RAxML v8.0.20; \citet{Stamatakis21012014}).
We used the fast bootstrap algorithm and performed 1000 bootstraps per tree inference to assess the topological support. 
The bootstrap algorithm used in RAxML is the Lazy Subtree Rearrangement (LSR) which consists in pruning one subtree from the tree and subsequently reinserting it to al neighbouring branches \citep{stamatakisa2008}.
Subtree Pruning and Reinserting methods (SPR) have been demonstrated as being better than others (e.g. Nearest Neighbouring Interchange - NNI) in recovering good bootstrap values \citep{salamin2003}.

\subsubsection{Bayesian}
The "best" tree and the "missing-data" trees were inferred using MrBayes v3.2.2 \citep{Ronquist2012mrbayes}.
We partitioned the data to treat the molecular part as a non-codon DNA partition and the morphological part as a multi-state morphological partition.
The molecular evolutionary history was inferred using the HKY model with a transition/transversion ratio of two \citep{douadycomparison2003} and a gamma distribution for the rate variation with four distinct categories (HKY + $\Gamma_4$).
For the morphological data, we used the Markov \textit{k} state model \citep{lewisa2001}, with equal state frequency and a unique overall substitution rate ($\mu$) with four distinct rates categories (M\textit{k} + $\Gamma_4$).
We chose these models to be consistent with the parameters used to generate the "complete" matrix.

Each Bayesian tree was estimated using two runs of four chains each for a maximum of 50$\times$$1^6$ generations.
We used the average standard deviation of split frequencies (ASDS) as a proxy to estimate the convergence of the chains and used a stop rule when the ASDS went below 0.01 \citep{Ronquist2012mrbayes}.
The effective sample size (ESS) was also checked on a random subsample of runs in each simulation to ensure that ESS $>>$ 200 \citep{drummond2006ess}.
For each run, we removed 25\% of the iterations as burnin.
We used the following priors for each tree (see \hyperref[supplementaries]{supplementaries}):
\begin{enumerate}
\item
the "true" tree’s topology as a starting tree (with a starting value for each branch length of 1),
\item
an exponential prior on the shape of the gamma distribution of $\alpha$=0.5 for both partitions
\item
and a transition/transversion ratio prior of 2 sampled from a strong beta distribution ($\beta$(80,40)).
\end{enumerate}

We used these priors to speed up the Bayesian process.
These priors biased the way the Bayesian process calculated the branch length by giving non-random starting points and boundaries for the parameters estimation process, however, in this study, we focused on the effect of missing data on the topology and not on the branch length.
Even using these priors, it took 65 CPU years to build 51 sets of 125 Bayesian trees (8 core nodes 2.30GHz clock speed).

%---------------------------------------------
%       Comparing Topologies
%---------------------------------------------

\subsection{Comparing topologies}
We compared the topology of the "missing-data" trees inferred from the matrices with missing data to the "best" tree to measure the effect of the three parameters $M_{L}$, $M_{F}$ and $M_{C}$.
Note that we only investigate differences in topology and not in branch length because the aim of this study is to look at the effect of missing data on the topology of trees inferred from matrices with living and fossil taxa and molecular and morphological characters.
To compare the topology of the resulting trees, we used two metrics to assess number of conserved taxa and clades position using respectively the Triple \citep{dobson1975triplets} and the Robison-Fould \citep{RF1981} distance.
We normalised the two metrics using the Normalised Tree Similarity index \citep{Bogdanowicz2012} to generalise our results for any \textit{n} number of taxa. The two metrics and the index are detailed below.

\subsubsection{Triple distance ($T_{x,y}$) \citep{dobson1975triplets}}
This metric measures the number of different subtrees of three clades between two given trees.
Each triplet can be written as $I_{ijk}$=(\textit{ijk}).
Where $I_{ijk}$ is equal to 0 if the the two triplets (\textit{ijk}) are the same in the two trees otherwise $I_{ijk}$ is equal to 1.
For any rooted binary tree there are only three possible combinations per triplets: ((\textit{j},\textit{k}),\textit{i});, ((\textit{i},\textit{k}),\textit{j}); and ((\textit{i},\textit{j}),\textit{k}); \citep{johnson1998}.
If the trees used are not fully binary, a fourth triplet combination is possible: (\textit{i},\textit{j},\textit{k});.
One can calculate $S_n$, the triplet distance between two trees as:
\begin{equation}
S_n = \sum_{ijk} I_{ijk}
\end{equation}
Where:
\begin{equation}
\sum_{ijk} = \binom{n}{4} = \frac{n!}{4!(n-4)!}
\end{equation}
And where \textit{n} is the number of taxa in both trees (modified from \citet{critchlowthe1996}).
If $S_n$=0, the trees are the same (i.e. no taxa as been displaced). When $S_n$ = $\binom{n}{4}$, the trees are the most different possible (i.e. every taxa as been displaced).
This metric therefore illustrates the amount of displayed taxa but is less sensitive to the placement of individual taxa and to taxa of highly uncertain placement (e.g. fossil taxa) than the Robinson-Foulds distance \citep{critchlowthe1996,johnson1998,wiensmissing2003} and can therefore be used as a proxy to estimate the robustness of the tree to flying taxa (see \hyperref[supplementaries]{supplementaries}).

\subsubsection{Robinson-Fould distance (\textit{RF}) \citep{RF1981}}
This metric measures the number of shared clades among two trees and therefore illustrates the number of exactly conserved groups among the trees.
The Robison-Fould distance (also called path difference) between two trees reflects the distance between the distributions of the tips among clades in the two trees \citep{RF1981} and can be expressed as following:
\begin{equation}
RF_{x,y} = N_{x} + N_{y} - 2C_{x,y}
\end{equation}
Where $C_{x,y}$ is the number of clades in common in the two trees.
The minimal value of \textit{C} is 1 if the two trees have the same \textit{n} taxa; the maximal value in \textit{C=n-2}.
This metric is more sensitive to taxa displacement than the triple metric (if one taxa gets out of a clade, then the clades are no longer considered as similar - \citet{critchlowthe1996,johnson1998,wiensmissing2003}) and therefore a low value will show a good clade conservation between two trees and a high value will show a bad recovery of common clades (see \hyperref[supplementaries]{supplementaries}).

\subsubsection{Normalised Tree Similarity \textit{NTS} \citep{Bogdanowicz2012}}
For any tree with \textit{n} taxa compared using a tree distance metric \textit{m}, $NTS_m$ represents the similarity score between the two trees given the expected distance between two random Yule trees with \textit{n} taxa.
Let $\bar{d}_{m,n}$\textit{(rand)} be the average distance between two random Yule trees with \textit{n} taxa and $d_{m,n}$\textit{(x,y)} the distance between the two trees \textit{x} and \textit{y} containing each \textit{n} taxa, then:
\begin{equation}
NTS_{m,n}(x,y)=\frac{\bar{d}_{m,n}(rand) - d_{m,n}(x,y)} {\bar{d}_{m,n}(rand)}
\end{equation}
\textit{NTS} ranges from 1 to -$\infty$.
For any \textit{m},\textit{n} when \textit{NTS}=1, the trees are the same;
when \textit{NTS}=0 the trees are not more different than expected by chance;
when \textit{NTS}$<$0, the trees are more different than expected by chance (see \hyperref[supplementaries]{supplementaries}).

$\newline$

We compared the "missing-data" tree to the "True" and the "Best" tree for each chain.
For the Maximum Likelihood trees we performed pairwise comparisons between the "True" and "Best" tree and the "missing-data" tree (where the "missing-data" tree is one of the 125 trees resulting from the 125 super matrices including various amount of missing data, see Table ~\ref{Tab_glossary}) for both the Robinson-Fould and the Triple metric.
We calculated the difference between the trees using the metrics described above by using the TreeCmp java script \citep{Bogdanowicz2012}.
For each metric, we normalized the value using the Normalised Tree Similarity scaled with the mean value of 1000 pairwise random tree comparison for the same metric and the same number of taxa \textit{n}=51 (see \hyperref[supplementaries]{supplementaries}).
We ran the comparison for every "missing-data" tree being one of the 125 combination of parameters values $M_L$, $M_F$ and $M_C$ in each chain resulting in 51 comparisons for every "missing-data" trees. 
We calculated the mode and the 50\% and 95\% confidence intervals from the resulting distribution using the hdrcde R package (v3.1 – \citet{hdrcde}).

Bayesian tree inference allows to account for the statistical uncertainty of a phylogenetic tree by not using an optimal criterion (c.f. Maximum Likelihood) and gives a tree posterior distribution (c.f. the likeliest tree in ML).
This method has the clear advantage of better dealing with error and uncertainty but is less practical for using the results (i.e. the phylogeny) for any further study (but see \citet{healyecology2014}).
To avoid this problem, people traditionally use the a consensus tree build on a majority rule in order to summarise a Bayesian tree posterior distribution leading to a single tree containing both topological and branch length information as well as support information (i.e. posterior probabilities, e.g. \citet{Ronquist2012mrbayes}).
However, using a Bayesian consensus tree has limitation especially if the resulting consensus tree is not well supported or resolved.
Because the metrics used in this study are used to measure variation in topology between two trees (i.e. taxa placement or clade position), comparing two Bayesian consensus trees is not optimal in picking topological difference signal.
For example, if one of the trees is not resolved at all (i.e. a star tree), comparing it to any other tree (even an other star tree) will not give useful results.
Therefore we used the entire Bayesian trees posterior distribution in order to perform the tree comparisons (i.e. the "Best" Bayesian tree posterior distribution vs. the "missing-data" Bayesian tree posterior distribution).

\subsubsection{Random Pairwise Bayesian Tree Comparison (RPBTC)}
We compared the distribution using a Random Pairwise Bayesian Tree Comparison (RPBTC) method.
This method consists in comparing a series of randomly selected pairs of trees from two posterior distributions (on from each distribution) and use the mode to summarize the resulting distribution as a proxy of the difference between the two trees in a Bayesian framework.
\begin{equation}
RPBTC_{X,Y}=Mo(d_{m,n}(X_{i1},Y_{j1}),d_{m,n}(X_{i2},Y_{j2}),d_{m,n}(X_{i3},Y_{j3}), ... ,d_{m,n}(X_{ik},Y_{jk}))
\end{equation}

Where \textit{X} and \textit{Y} are two Bayesian tree posterior distribution;
 $d_{m,n}$ is the pairwise difference for any metric \textit{m} and \textit{n} taxa between $X_{i}$ and $Y_{j}$ which are two single trees randomly sampled respectively from \textit{X} and \textit{Y}
 and $Mo(d_{m,n}(X_{i1},Y_{j1}),d_{m,n}(X_{i2},Y_{j2}),d_{m,n}(X_{i3},Y_{j3}), ... ,d_{m,n}(X_{ik},Y_{jk}))$ is the mode of the pairwise difference repeated \textit{k} times.
We used the mode to summarize the distributions because the value that is the most represented in the distribution reflects more accurately what a consensus tree represents (the topology the most represented in the posterior distribution).
Also, the mode represents a value that is present in the distribution (which, depending on the metric, can be a series of discrete values) in contrast to the other distribution summary metrics, such as the mean or the median, which can take values that are not actually present in the distribution.
For example, if comparing two posterior tree distribution, we obtain difference values of 10, 10, 10, 5, 5 and 1, using the mean (6.8) or the median (7.5) gives a values that actually doesn't exist

In this study, we calculated RPBTC for both metrics (RF and Triple distance) for each pairs of Bayesian tree distributions (i.e. the "Best" Bayesian tree posterior distribution vs. the "missing-data" Bayesian tree posterior distribution) with 1000 random pairwise comparisons (\textit{k}).
Because the RPBTC uses random pairwise comparisons, there is a chance of comparing always trees that have the biggest or the smallest difference between two distributions, either deflating or inflating the RPBTC difference value.
However, using 1000 random pairwise comparisons makes the difference stable (i.e. if repeated independently, no difference is detected - see \hyperref[supplementaries]{supplementaries}).

$\newline$

We calculated the NTS for the "Best" Bayesian tree vs. "missing-data" Bayesian tree comparison for both RF and Triple metric using the mode of the RPBTC for each chain resulting in a distribution of 51 modes per comparison.
We then compared the NTS of the Bayesian trees to the NTS of the ML trees for both metrics for each comparison.
This resulted in a distribution of 51 NTS values for the Bayesian trees and 51 NTS values for the ML trees for each tree comparison for which we calculated the mode and the 50\% and 95\% confidence intervals (see \hyperref[supplementaries]{supplementaries} - Fig. ~\ref{Fig_Compare}).

\begin{figure} % Modify the figure
\centering
\includegraphics[keepaspectratio=true]{Figures/TEM_Fig_TreeCmp-BW.pdf}
\caption{Tree comparison protocol.
From each of the 125 parameters combination ($M_L$, $M_F$ and $M_C$), we inferred a tree in Baysian or Maximum likelihood framework.
We then compared this tree with or without fossil/living taxa to the "best/true" tree.
In the Bayesian framework, 1000 trees from the posterior distribution where used for each comparison.}
\label{Fig_Compare}
\end{figure}

\subsubsection{Statistical difference between tree topologies}
To assess the difference between the tree topologies, whether they were inferred in a Bayesian or a ML framework, we performed a group comparison test for each set of configurations (e.g. effect of $M_L$ in Bayesian inference, combined effect of $M_F$ and $M_C$ in ML inference, etc...).
We used the pairwise tree comparisons as factors (i.e. the "Best" tree vs. one of each of the 125 "missing-data" trees) and the values of the 51 replicates from the pairwise comparison as a response variable.
The 51 replicates values were either the replicates of the 51 pairwise tree comparisons in ML or the modes of the 51 RPBTC posterior distributions in Bayesian.
When we found a significant difference between the groups, we performed a pairwise comparison test to analyse which pairs of groups differ from the others.
Depending on whether the data was normal with variance homoscedasticity or not (controlled by performing respectively a shapiro.test and a a bartlett.test in R\{stats\}), we used respectively parametric or non-parametric tests for the group comparison and the pairwise comparisons as suggested by \citet{ruxtontime2008} (Table ~\ref{Stats_test}). %link broken

\begin{table}[ht]
\caption{Statistical tests}
\centering
\begin{tabular}{lrll}
  \hline
   & test & statistics & code\{package\} \\ [0.5ex]
  \hline
  Groups comparisons & ANOVA & parametric & anova\{stats\} \\ [0.5ex]
                     & Kruskal-Wallis & non-parametric & kruskal.test\{stats\} \\[0.5ex]
  \hline
  Pairwise comparisons & Tukey HSD & parametric & TukeyHSD\{stats\} \\ [0.5ex]
                       & Nemenyi-Damico-Wolfe-Dunn & non-parametric & kruskalmc\{pgirmess\} \\ [0.5ex]
  \hline
\end{tabular}
\label{Stats_test}
\end{table}

\subsection{Effect of missing molecular characters for fossil taxa} %Awfull name.
To assess the effect of missing molecular characters for fossil taxa in a Bayesian framework, we split the "complete" matrix in sub-matrices containing no missing data at all:
\begin{enumerate}
\item
A first containing both molecular and morphological data for living taxa only (hereafter called the "horizontal" sub-matrix, Table ~\ref{Tab_glossary});
\item
A second one containing morphological data for both living and fossil taxa (hereafter called the "vertical" sub-matrix, Table ~\ref{Tab_glossary});
\item
A third one containing morphological data for living taxa only (hereafter called the "corner" sub-matrix, Table ~\ref{Tab_glossary});
\end{enumerate}
We reran the Bayesian tree inference on the different sub-matrices with no missing data in the same way as described above.
We then compared the resulting tree posterior distribution to itself (in the same way described above) to assess the ability to recover topology for each simulation when no missing data was involved in the phylogenetic inference process.

%---------------------------------------------
%       Empirical data - TO DO
%---------------------------------------------

\subsection{Empirical data} %Still not done!
We also compared the results obtained from simulated data by using \citet{ronquista2012} empirical data.
The matrix contains 67 living species plus one outgroup and 45 fossil species of Hymenopteras with 5097 molecular characters and 354 morphological characters.
From the 68 living species used in the matrix, only 66 had molecular data, we therefore treated these 66 taxa as "living" taxa and all the other 47 as "fossil" taxa.
We treated the matrix in the exact same way as described in step 2 and 3 resulting in 125 matrices with various amount of missing data and the same number of Maximum Likelihood and Bayesian trees.
We used the same settings as for the simulated data in the Maximum Likelihood framework.
For the Bayesian inferences however, we didn’t used any priors except that we provided a starting tree with the topology of the 68 living species (topology with the highest posterior probability from non-clock analysis - \citet{ronquista2012}).
Contrary to \citet{ronquista2012} analysis, we didn’t performed any clock analysis since we were only interested in the topology of the inferred tree and not the branch length.


%---------------------------------------------
%
%       RESULTS
%
%---------------------------------------------


\section{Results}
%Do the check for L00F00C00
\subsection{Building the trees}
Bayesian chains converged (SD $<$ 0.01 and ESS $>>$ 200 for each tree).
Good a quick convergence when using the sub-matrices especially for the "horizontal" sub-matrices.
Good enough phylogenetic signal.

\subsection{Comparing topologies}
Did it worked? See sub-matrices

\subsection{Effect of the missing data parameters}

\subsubsection{Effect of $M_L$}
In ML...(see Table ~\ref{Group_results})

Topological recovery was bad in Bayesian, the NTS score was only slightly above 0 which means that the mode of the RPBTC was only slightly different than expected by chance.
However, it seems that there is a small difference depending on the metric used.
For the Triple metric there is no significant effect of the $M_L$ parameter, in other words, if living taxa are removed, there is no effect on the placement of "flying" taxa.
For the Robinson-Fould metric, there is a significant effect of the $M_L$ leading small decrease in topological recovery when $M_L$ increases.
In other words, if living taxa are removed, clades are less conserved (see Table ~\ref{ML_results}).
However, in both cases, the effect of the parameter $M_L$ is still minimal compared to trees generated in a ML framework since the NTS score was only slightly higher than expected by chance (see Table ~\ref{Group_results}).

\subsubsection{Effect of $M_F$}
In ML...(see Table ~\ref{Group_results})

Similarly than for the effect of $M_L$, Bayesian tree recovery was only slightly better than expected by chance.
And in the same way as described above, there seems to be a small difference depending on the metric used.
There is no significant effect of the $M_F$ parameter when using the Triplet metric but the effect is significant when using the Robinson-Fould metric (see Table ~\ref{ML_results}).
However, as mentioned previously the effect of the parameter $M_F$ is still minimal in a Bayesian framework (see Table ~\ref{Group_results}).

\subsubsection{Effect of $M_C$}
In ML...(see Table ~\ref{Group_results})

The effect of $M_C$ in Bayesian framework is similar as the one described for $M_L$ and $M_F$: tree recovery is only slightly better than expected by chance and only the Robinson-Fould metric shows a significant effect of the $M_C$ parameter on tree recovery (see Table ~\ref{ML_results} and ~\ref{Group_results}).
\subsubsection{Combined effect}

In ML...(see Table ~\ref{Group_results})

In Bayesian, same pattern as above.
No particular effect of data combination, but more an effect of the overall amount of data (see Table ~\ref{Group_results}).

$\newline$

The ability of recovering the best topology in maximum likelihood method is function of the amount of data missing.
However, in Bayesian, topology is badly (if not at all) recovered independently of the amount of data missing.


%Add the global heat plot figures
\begin{table}
\caption{Group difference per configuration set}
\centering
\begin{tabular}{rlrlccc}
    \hline
    set & metric & framework & parametric & $stats^1$ & df & p.value \\
    \hline
    $M_L$ & Robinson-Fould & ML & bla & bla & bla & bla \\
          &                & Bayesian & no & 65.62 & 4.00 & \textbf{0.00} \\
          & Triples & ML & bla & bla & bla & bla \\
          &         & Bayesian & no & 4.56 & 4.00 & 0.34 \\
    $M_F$ & Robinson-Fould & ML & bla & bla & bla & bla \\
          &                & Bayesian & no & 85.61 & 4.00 & \textbf{0.00} \\
          & Triples & ML & bla & bla & bla & bla \\
          &         & Bayesian & no & 3.01 & 4.00 & 0.56 \\ 
    $M_C$ & Robinson-Fould & ML & bla & bla & bla & bla \\
          &                & Bayesian & no & 64.23 & 4.00 & \textbf{0.00} \\
          & Triples & ML & bla & bla & bla & bla \\
          &         & Bayesian & no & 26.13 & 24.00 & 0.35 \\
    $M_L+M_F$ & Robinson-Fould & ML & bla & bla & bla & bla \\
              &                & Bayesian & no & 317.97 & 24.00 & \textbf{0.00} \\
              & Triples & ML & bla & bla & bla & bla \\
              &         & Bayesian & no & 3.01 & 4.00 & 0.56 \\ 
    $M_L+M_C$ & Robinson-Fould & ML & bla & bla & bla & bla \\
              &                & Bayesian & no & 290.10 & 24.00 & \textbf{0.00} \\
              & Triples & ML & bla & bla & bla & bla \\
              &         & Bayesian & no & 22.19 & 24.00 & 0.57 \\ 
    $M_F+M_C$ & Robinson-Fould & ML & bla & bla & bla & bla \\
              &                & Bayesian & no & 385.96 & 24.00 & \textbf{0.00} \\
              & Triples & ML & bla & bla & bla & bla \\
              &         & Bayesian & no & 20.23 & 24.00 & 0.68 \\
    $M_L+M_F+M_C$ & Robinson-Fould & ML & bla & bla & bla & bla \\
                  &                & Bayesian & no & 1167.40 & 124.00 & \textbf{0.00} \\
                  & Triples & ML & bla & bla & bla & bla \\
                  &         & Bayesian & no & 115.86 & 124.00 & 0.69 \\ 
    \hline
\label{Group_results}
\end{tabular}

$^1$ F-value for parametric tests and Kruskal Wallis $chi^2$ for non parametric tests.
\end{table}

\begin{table}
\caption{$M_L$ non-parametric pairwise difference for Robinson-Fould metric}
\centering
\begin{tabular}{c|ccccc}
    \hline
              & $M_L$00\% & $M_L$10\% & $M_L$25\% & $M_L$50\% & $M_L$75\% \\
    \hline
    $M_L$00\% & - & FALSE & \textbf{TRUE} & \textbf{TRUE} & \textbf{TRUE}\\
    $M_L$10\% & & - & FALSE & \textbf{TRUE} & \textbf{TRUE} \\
    $M_L$25\% & & & - & FALSE & \textbf{TRUE} \\
    $M_L$50\% & & & & - & FALSE \\
    $M_L$75\% & & & & & - \\
    \hline
\end{tabular}
\centering
\begin{tabular}{rrrl}
 pairwise comparison & obs.dif & critical.dif & difference \\ 
  \hline
  $M_L$00\%-$M_L$10\% & 17.05 & 41.00 & FALSE \\ 
  $M_L$00\%-$M_L$25\% & 43.79 & 41.00 & TRUE \\ 
  $M_L$00\%-$M_L$50\% & 77.32 & 41.00 & TRUE \\ 
  $M_L$00\%-$M_L$75\% & 103.99 & 41.00 & TRUE \\ 
  $M_L$10\%-$M_L$25\% & 26.75 & 41.00 & FALSE \\ 
  $M_L$10\%-$M_L$50\% & 60.27 & 41.00 & TRUE \\ 
  $M_L$10\%-$M_L$75\% & 86.94 & 41.00 & TRUE \\ 
  $M_L$25\%-$M_L$50\% & 33.53 & 41.00 & FALSE \\ 
  $M_L$25\%-$M_L$75\% & 60.20 & 41.00 & TRUE \\ 
  $M_L$50\%-$M_L$75\% & 26.67 & 41.00 & FALSE \\ 
   \hline
\end{tabular}
\label{ML_results}
\end{table}

\begin{table}
\caption{$M_F$ non-parametric pairwise difference for Robinson-Fould metric in Bayesian framework}
\centering
\begin{tabular}{c|ccccc}
    \hline
              & $M_F$00\% & $M_F$10\% & $M_F$25\% & $M_F$50\% & $M_F$75\% \\
    \hline
    $M_L$00\% & - & FALSE & FALSE & \textbf{TRUE} & \textbf{TRUE}\\
    $M_L$10\% & & - & FALSE & \textbf{TRUE} & \textbf{TRUE} \\
    $M_L$25\% & & & - & FALSE & \textbf{TRUE} \\
    $M_L$50\% & & & & - & FALSE \\
    $M_L$75\% & & & & & - \\
    \hline
\end{tabular}
\centering
\begin{tabular}{rrrl}
 pairwise comparison & obs.dif & critical.dif & difference \\ 
  \hline
  $M_F$00\%-$M_F$10\% & 16.03 & 41.00 & FALSE \\ 
  $M_F$00\%-$M_F$25\% & 38.50 & 41.00 & FALSE \\ 
  $M_F$00\%-$M_F$50\% & 61.52 & 41.00 & TRUE \\ 
  $M_F$00\%-$M_F$75\% & 95.52 & 41.00 & TRUE \\ 
  $M_F$10\%-$M_F$25\% & 22.47 & 41.00 & FALSE \\ 
  $M_F$10\%-$M_F$50\% & 45.49 & 41.00 & TRUE \\ 
  $M_F$10\%-$M_F$75\% & 79.49 & 41.00 & TRUE \\ 
  $M_F$25\%-$M_F$50\% & 23.02 & 41.00 & FALSE \\ 
  $M_F$25\%-$M_F$75\% & 57.02 & 41.00 & TRUE \\ 
  $M_F$50\%-$M_F$75\% & 34.00 & 41.00 & FALSE \\ 
   \hline
\end{tabular}
\label{MF_results}
\end{table}

\begin{table}
\caption{$M_C$ non-parametric pairwise difference for Robinson-Fould metric in Bayesian framework}
\centering
\begin{tabular}{c|ccccc}
    \hline
              & $M_C$00\% & $M_C$10\% & $M_C$25\% & $M_C$50\% & $M_C$75\% \\
    \hline
    $M_C$00\% & - & FALSE & FALSE & \textbf{TRUE} & \textbf{TRUE}\\
    $M_C$10\% & & - & FALSE & \textbf{TRUE} & \textbf{TRUE} \\
    $M_C$25\% & & & - & FALSE & \textbf{TRUE} \\
    $M_C$50\% & & & & - & FALSE \\
    $M_C$75\% & & & & & - \\
    \hline
\end{tabular}
\centering
\begin{tabular}{rrrl}
 pairwise comparison & obs.dif & critical.dif & difference \\ 
  \hline
  $M_C$00\%-$M_C$10\% & 16.03 & 41.00 & FALSE \\ 
  $M_C$00\%-$M_C$25\% & 38.50 & 41.00 & FALSE \\ 
  $M_C$00\%-$M_C$50\% & 61.52 & 41.00 & TRUE \\ 
  $M_C$00\%-$M_C$75\% & 95.52 & 41.00 & TRUE \\ 
  $M_C$10\%-$M_C$25\% & 22.47 & 41.00 & FALSE \\ 
  $M_C$10\%-$M_C$50\% & 45.49 & 41.00 & TRUE \\ 
  $M_C$10\%-$M_C$75\% & 79.49 & 41.00 & TRUE \\ 
  $M_C$25\%-$M_C$50\% & 23.02 & 41.00 & FALSE \\ 
  $M_C$25\%-$M_C$75\% & 57.02 & 41.00 & TRUE \\ 
  $M_C$50\%-$M_C$75\% & 34.00 & 41.00 & FALSE \\ 
   \hline
\end{tabular}
\label{MC_results}
\end{table}

%---------------------------------------------
%
%       DISCUSSION
%
%---------------------------------------------


\section{Discussion}

\subsection{Building the trees}
Hard to simulate morpho characters but see Pagel, April Wright, etc...

\subsection{Comparing topologies}
Hard to compare Bayesian tree posterior distributions topologies but gives a good idea.

\subsection{ML vs Bayesian}
Missing data is not a problem as long as random. However, in TEM, n.fossils*n.molecular data is missing and not random.
Non optimal criterion approach (i.e. Bayesian) worse because of the number of considered "less"-likely trees (flat maximum likelihood landscapes).
However, ML still good performances and can be used to fix topology.

\subsection{Effect of missing data}
Parameter blabla seems to be influencing the topology the most.
But in practice, $M_L$ is the only one realistically solvable. Go to museums!

What about the "best" trees with fairly bad bootstrap support values in ML?
This is probably due to the low number of characters used in the simulation.
The number of morphological characters ($10^2$) is in the same order of magnitudes of empirical studies (cite) but decreases unrealistically with the $M_C$ parameter. %Extract the mean number of characters per matrices from the 100 random ones
The number of molecular characters ($10^3$) is at least two orders of magnitude lower than in empirical studies that can reach $10^6$ characters (cite).

Two different matrices from the "True" tree gives two different phylogenetic interpretations (i.e. seqGen and ape performs badly at generating a data matrix from a tree)?
That seems to not really be a problem since we are interested in recovering topology with missing data, not assessing if phylogenetic matrices generation software performs correctly.
Also, the "noise" that might be created by this potential "bad" phylogenetic matrices generation software is probably diminished by the number of replicates (i.e. 50 chains).

Different phylogenetic signal between the morphological and the molecular matrix?
That is not really a problem since the phenotypic evolution is not necessary correlated to the genotypic one.
Also it is important to note that both information are not exactly the same.
The main difference between molecular and morphological characters is that, because of historical and practical reasons, a morphological character matrix do no contains autapomorphies and invariant characters which are usually more common than synampomorphies in molecular character matrices (cite).

What about homoplasy in the morphological matrix?
Here, we made the assumption that theoretically, morphological characters are randomly distributed on an organism however it seems clear that empirical morphological data does not act randomly
If one states that morphological characters accumulate through time in the same way as the majority of the molecular characters (neutral theory, cite) then, homoplasic characters are expected to appear randomly through time.
Therefore, homoplasy is expected to be more important (by chance) in bigger morphological matrices \citep{davalosintegrating2014}.
After a certain amount of morphological characters, adding new ones adds homoplasy \citep{wagner2000}.

%Potential reviewers comments?
%What about it when you vary the proportion of fossil taxa?


%---------------------------------------------
%
%       CONCLUSION
%
%---------------------------------------------


\section{Conclusion}
A Bayesian approach fails to recover accurate topology in a total evidence approach framework, whatever the amount of missing data.
We think this failure is due to the intrinsic structure of the Total Evidence matrices that includes a vast amount of not randomly distributed missing data (i.e. the molecular part of the matrix for the fossil taxa).
This missing data is equal to the number of fossil taxa $\times$ the number of molecular characters leading to a vast amount of likely trees to be sampled in the Bayesian mcmc tree building.
However, one can use a Maximum Likelihood approach to obtain the likeliest topology and to use this one as a fixed.


%---------------------------------------------
%
%       ...
%
%---------------------------------------------


\section{Acknowledgements}
We would like to thank Trevor Hodkinson and Andrew Jackson %Also Gavin Thomas, Emmanuel Douzery and Frédéric Delsuc if not authors.
for his useful comments on the simulation protocol and Paddy Doyle for the assistance on using the computer cluster.
All calculations were performed on the Lonsdale cluster maintained by the Trinity Centre for High Performance Computing.
This cluster was funded through grants from Science Foundation Ireland.

%BIBLIOGRAPHY
 % The \cite command functions as follows:
 %   \citet{key} ==>>                Jones et al. (1990)
 %   \citet*{key} ==>>               Jones, Baker, and Smith (1990)
 %   \citep{key} ==>>                (Jones et al., 1990)
 %   \citep*{key} ==>>               (Jones, Baker, and Smith, 1990)
 %   \citep[chap. 2]{key} ==>>       (Jones et al., 1990, chap. 2)
 %   \citep[e.g.][]{key} ==>>        (e.g. Jones et al., 1990)
 %   \citep[e.g.][p. 32]{key} ==>>   (e.g. Jones et al., p. 32)
 %   \citeauthor{key} ==>>           Jones et al.
 %   \citeauthor*{key} ==>>          Jones, Baker, and Smith
 %   \citeyear{key} ==>>             1990

\bibliographystyle{sysbio} %don't write the suffix
\bibliography{References} %don't write the suffix

\section{Supplementaries}
\label{supplementaries}
\subsection{Morphological characters states}
In order to obtain a realistic probabilistic value for of \textit{k} characters states for each simulated morphological character, we downloaded 100 random morphological characters (with more than 100 characters each) from TreeBASE database (http://treebase.org/) published between 1985 and 2013 and covering 19 taxomomic classess (Chordata, Arthropoda, Annelida, Angiosperm, Gymnosperm and Pteridophyta).
We selected a total of 22563 characters ranging from 2 to 10 states.
We calculated the proportion of characters with 2, 3, 4, 5, 6, 7, 8, 9 or 10 states.
We then sampeled 22563 \textit{k} values between 2 and 10 with the same proportion of characters from the empirical data.
We then used a simple t-test to check if our simulation was equal to the empirical data.
In this study, we only simulated characters with 2 or 3 states because of the high proportion of ordered characters ecountered on characters with more than 3 states and the difficulties of simulate biologicaly sensible ordered characters.

\begin{figure}
\centering
\includegraphics[keepaspectratio=true]{Figures/TEM_Fig-AppendixCharacters.pdf}
\caption{Character states distribution in empirical matrices. %Title?
Characters states number distribution extracted from 100 random morphological matrices downloaded from RreeBase.}
\label{Fig_AppendixCharacters}
\end{figure}

\subsection{Tree Building Software settings}

\subsubsection{Maximum Likelihood - RAxML v8.0.20 \citep{Stamatakis21012014}} \\
Model: \\
Molecular data: \\
GTR + $\Gamma_4$ (-m GTRGAMMA) \\
Morphological data: \\
Mk + $\Gamma_4$ (-K MK) \\
Support: \\
Rapid Boostrap algorithm (LSR), 1000 replicates \\

\subsubsection{Bayesian - MrBayes v3.0.2 \citep{Ronquist2012mrbayes}} \\
Priors: \\
Molecular data: \\
rates distribution shape ($\alpha$) = 0.5 \\
Transition/Transversion ratio = 2 ($\beta$(80,40)) \\
Starting tree: "True" tree topology with each branch length = 1 \\
Morphological data: \\
rates distribution shape ($\alpha$) = 0.5 \\
Models: \\
Molecular data: HKY + $\Gamma_4$ \\
Morphological data: Mk + $\Gamma_4$ \\
MCMC: \\
2 runs \\
4 chains per run \\
generations < 50$\times$$1^6$ \\
sample frequency = 1050$\times$$1^3$ \\
ASDS diagnosis frequency = 50$\times$$1^3$ \\
ASDS $<$ 0.01 \\
ESS $>>$ 200 \\
Burnin = 25\%

\subsection{Triplets metric details ($T_{x,y}$)}
Each triplet can be written as $I_{ijk}$=(\textit{ijk})). Where $I_{ijk}$ is equal to 0 if the the two triplets (\textit{ijk}) are the same in the two trees otherwise $I_{ijk}$ is equal to 1.
For any rooted tree there are only four possible combinations per triplets: ((\textit{j},\textit{k}),\textit{i});, ((\textit{i},\textit{k}),\textit{j}); and ((\textit{i},\textit{j}),\textit{k}); and (\textit{i},\textit{j},\textit{k}); \citep{johnson1998}.
One can calculate $S_n$, the triplet distance between two trees as:
\begin{equation}
S_n = \sum_{ijk} I_{ijk}
\end{equation}
Where:
\begin{equation}
\sum_{ijk} = \binom{n}{4} = \frac{n!}{4!(n-4)!}
\end{equation}
And where n is the number of taxa in both trees (modified from \citet{critchlowthe1996}).
When all triplets across the two trees are the same, $S_n$ is equal to 0 and when all the triplets are different $S_n$ is equal to $\binom{n}{4}$.
Because the possible number of triplets per clade is a finite number, the probability of two random trees with the same n taxa to have the same triplet is:
\begin{equation}
P({I_{ijk}}=0) = \frac{1}{4}
\end{equation}
Therefore one can calculate the probability of two random trees having the same triplets: 
\begin{equation}
P({S_{n}}=0) = \sum_{ijk} P_{I_{ijk}=0}
\end{equation}
\begin{equation}
P({S_{n}}=0) = \frac{n!}{4(3!(n-3)!}
\end{equation}
And in the same way:
\begin{equation}
P({S_{n}}=1) = \frac{3n!}{4(3!(n-3)!}
\end{equation}

\subsection{RF metric details}
The RF distance (or path difference) between two trees reflects the distance between the distributions of the tips among clades in the two trees \citep{RF1981} and can be expressed as following:
\begin{equation}
RF_{x,y} = N_{x} + N_{y} - 2C_{x,y}
\end{equation}
Where $C_{x,y}$ is the number of clades in common in the two trees. 
The minimal value of \textit{C} is equal to 1 if the two trees have the same n taxa;
the maximal value in \textit{C}=\textit{n}-2.
For a fully unresolved tree (star tree) \textit{N}=1 and for a fully resolved tree (binary tree) \textit{N}=\textit{n}-2.
The minimal and maximal topological distance for \textit{n} taxa is:
\begin{equation}
RF_{min} = 1 + 1 - 2C_{x,y}
\end{equation}
And:
\begin{equation}
RF_{max} = 2(n-2)-2
\end{equation}
One can then rescale \textit{RF.scaled} by using the maximal and minimal value for any \textit{n} taxa:
\begin{equation}
RF.scaled_{x,y} = \frac{RF_{x,y}-RF_{max}}{RF_{max}}
\end{equation}
This metric is more sensitive to taxa displacement than the Triplet distance \citep{critchlowthe1996,johnson1998,wiensmissing2003} and therefore a low value will show a good clade conservation between two trees and a high value will show a bad recovery of common clades.

\subsection{Tree comparisons}
\subsubsection{Random tree comparison scaling}
We used the comparison of 1000 random trees to obtain the mean comparison value $\bar{d}_{m,n}$\textit{(rand)} for the NTS metric.
We randomly generated two sets of 1000 trees of \textit{n} taxa using the rmtree function of ape package (v3.0-11 \citet{paradisape:2004}) that generates a given number of random Yule trees.
We calculated the $\bar{d}_{m,n}$\textit{(rand)} value using an approach similar to the RPCBTC (described below) by performing 1000 random pairwise comparisons using the TreeCmp java script \citep{Bogdanowicz2012}.

\subsubsection{Random Pairwise Bayesian Tree Comparison (RPBTC)}
We assessed the power of the Random Pairwise Bayesian Tree Comparison (RPBTC) method by comparing 1000 random trees from a posterior distribution trees set to another 1000 random trees from the same posterior distribution trees set.
We repeated this 100 times independently using the same posterior distribution trees set each time resulting in 100 replicates of the same posterior distribution trees set compared 1000 times.
We used an anova to test if there was no significant difference between the replicates so that the RBTC can be replicated.
We applied this protocol on a poorly resolved tree (Low Score), a resolved tree with low support value (Medium Score) and a resolved tree with high support values (High Score).
Results are available in table ~\ref{RPBTC_testing}. %link broken

\begin{table}[ht]
\caption{Group comparison results: difference between 100 replicates using the RPBTC method} %test with TreeCmp.anova
\centering
\begin{tabular}{rllrrrr}
  \hline
  Tree.Type & Used.metric & Replicates & Df & F.value & p.value \\ 
  \hline
  Low Score & RF & 100.00 & 99.00 & 0.74 & 0.98 \\ 
  Low Score & Tr & 100.00 & 99.00 & 0.97 & 0.58 \\ 
  Medium Score & RF & 100.00 & 99.00 & 0.64 & 1.00 \\ 
  Medium Score & Tr & 100.00 & 99.00 & 0.45 & 1.00 \\ 
  High Score & RF & 100.00 & 99.00 & 0.20 & 1.00 \\ 
  High Score & Tr & 100.00 & 99.00 & 0.37 & 1.00 \\ 
  \hline
\end{tabular}
\end{table}
\label{RPBTC_testing}

\subsection{Codes}
All codes are available at: $https://github.com/TGuillerme/Total\_Evidence\_Method-Missing\_data/tree/master/Functions$

%END
\end{document}